packages/style-engine/class-wp-style-engine.php

```diff
@@ -148,13 +148,19 @@ class WP_Style_Engine {
 					'individual' => 'padding-%s',
 				),
 				'path'          => array( 'spacing', 'padding' ),
+				'css_vars'   => array(
+					'space' => '--wp--preset--spacing-size--$slug',
```

Comment:
Apologies for not noticing/asking about this sooner, but I was wondering if this should be `spacing` instead of `space` for consistency?

Reply:
yeh, that makes sense, done.

Reply:
Thanks for taking care of that! 🙇